packages/plot/src/interactors/util/pointerIsDown.js

```diff
@@ -0,0 +1,6 @@
+export let pointerIsDown = false;
+
+if (typeof document !== 'undefined') {
```

Comment:
Hmm. On my TODO list is to make Mosaic work with server-side rendering, where we can pass in a document as part of the API context. I'd like to avoid random references that assume `document` is defined in the global scope only. I'll think about alternatives...

MouseEvent is way easier as it includes which buttons are down.